app/src/main/java/com/dcrandroid/fragments/SendFragment.java

```diff
@@ -236,18 +236,23 @@ public void onClick(View v) {
                 }
             }
         });
-        if(Integer.parseInt(util.get(Constants.CURRENCY_CONVERSION, "-1")) != 0) {
+        if(Integer.parseInt(util.get(Constants.CURRENCY_CONVERSION, "-1") ) != 0 && util.get(Constants.CURRENCY_CONVERSION, "-1")  != null ){
```

Comment:
This is not the fix. The bug is coming from the `-1` default value used to get the preference, it should be 0 instead.